feat(changelog): containers-added-serverless-containers-and-vpc 2025-07-21 (#5304)
scaleway/docs-content: changelog/july2025/2025-07-21-containers-added-serverless-containers-and-vpc.mdx

---
title: Serverless Containers and VPC
status: added
date: 2025-07-21
category: serverless
product: containers
---

> You can now connect your Serverless Containers with Private Network, allowing secured access between Serverless to resources inside the same Private Network like databases. 

Reviewer: ```suggestion
You can now connect your Serverless Containers to a Private Network, allowing secure access between Serverless and resources inside the same Private Network, such as Managed Databases, Instances and more.
```

> [Read more](https://www.scaleway.com/en/docs/serverless-containers/how-to/use-private-networks/) about attaching a Private Network to a Serverless Container.

Reviewer: ```suggestion
[Read more](/serverless-containers/how-to/use-private-networks/) about attaching a Private Network to a Serverless Container.
```
